tests/unit-tests/api/system-status.php

```diff
@@ -376,4 +376,32 @@ public function test_execute_system_tool_add_order_indexes() {
 		$this->assertNotEmpty( get_post_meta( $order2->get_id(), '_shipping_address_index', true ) );
 
 	}
+
+	/**
+	 * Test system_status _filter query parameter.
+	 */
+	public function test_get_system_status_info_filtered() {
+		wp_set_current_user( $this->user );
+		$query_params = array(
+			'_fields' => 'theme,settings,nonexisting',
+		);
+		$request      = new WP_REST_Request( 'GET', '/wc/v2/system_status' );
```

Comment:
I think we should be testing the `v3` api endpoints in these. 

Also GitHub removes all your line breaks when copy+pasting for some reason, so you've lost those in this PR. :) That doesn't really require fixing though unless you want; just a heads-up.